docs/api/presenter.md

```diff
@@ -41,7 +41,7 @@ class PlanetsPresenter extends Presenter {
 
   getModel (props, state) {
     return {
-      planets: data => data.planets
+      planets: state => state.planets
```

Comment:
Ah, yeah sorry myself, this change is definitely not ideal. We can probably leave this as is for now.